src/pages/home/HeaderView.js

```diff
@@ -75,7 +75,7 @@ const HeaderView = (props) => {
     const isChatRoom = ReportUtils.isChatRoom(props.report);
     const isPolicyExpenseChat = ReportUtils.isPolicyExpenseChat(props.report);
     const isTaskReport = ReportUtils.isTaskReport(props.report);
-    const reportHeaderData = isTaskReport && !_.isEmpty(props.parentReport) ? props.parentReport : props.report;
+    const reportHeaderData = isTaskReport && !_.isEmpty(props.parentReport) && !isThread ? props.parentReport : props.report;
```

Comment:
Same question as above.. How can a report have `parentReport` and not be a thread?

Reply:
replied with the same answer above 🙂

Reply:
I don't think the same answer applies here :sweat_smile: In this one we already have a parent report but somehow not a thread?

If we have a parent report, then this report has parentReportID and parentReportActionID so it must be a thread.

Reply:
oh lol this is the inverse. I just saw same words and thought it was the same question 🙈 

> How can a report have parentReport and not be a thread?

I think there are mixed terms here between two projects so its a little confusing. Let me try to explain,
For threads, in onyx we will have,
- `reportActions_<parentReportID>`
- `report_<childReportID>`
- `reportActions_<childReportID>`

So the user may or may not have `report_<parentReportID>` depending on their access.

okay so with that said. I think the code here might break tasks, because tasks aren't all tasks also a thread. Georgia/stites can you double check this logic doesn't break for tasks?

Reply:
<img width="1176" alt="2023-05-11_18-35-14" src="https://github.com/Expensify/App/assets/42391420/9a114c70-22b3-417a-949c-c6c2eb8d8913">
Is this what you were worried about? Looks like threads don't have titles in the assign task menu

Reply:
I'm not really sure what tasks are supposed to look like, but after picking the top blank chat it looks like this
<img width="1173" alt="2023-05-11_18-35-59" src="https://github.com/Expensify/App/assets/42391420/6aee0a81-2d80-48d6-833b-a3d712a1c727">

Reply:
well the screenshot you shared looks like a different bug, maybe also related to this issue. 
but the comment here is about header view so i don't think it related to "share somewhere" screen.

Reply:
I just checked the header for tasks. On main it is,
![image](https://github.com/Expensify/App/assets/14356145/15fa5c79-e65b-4211-8d04-490b9eb4f298)

In this PR it is, (so i think this PR broke it)
![image](https://github.com/Expensify/App/assets/14356145/a7cd1053-46a7-4dce-9b14-03d1fcaf9360)

The UI also seems broken for tasks by showing a message at the bottom.

Reply:
Okay i'll think of how to change this check

Reply:
Fixed the one this comment is about, i will look into the other one